### PR TITLE
Focus states IA and content updates

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _docs/style/focus-states.md [hunk 1]
```diff
@@ -3,29 +3,46 @@ title: Focus States
...
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct focus states:
+
+**CHECK WITH RYAN TO SEE IF ANY ACCESSIBILITY BEST PRACTICE NUGGETS MIGHT BE RELEVANT**
+
+- Apply focus state to any element that can be interacted with using a mouse, keyboard, touch, or voice command.
+- xxxx.
```
Comment (chri2547) [inline _docs/style/focus-states.md]: A disabled component cannot be in focus.

File: _docs/style/focus-states.md [hunk 2]
```diff
@@ -3,29 +3,46 @@ title: Focus States
...
 
-## Introduction
-
 <div class="hxRow" markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Focus is received by interactive components when the user clicks on them.  Focus is also applied to clickable elements when the user presses the tab key to traverse the UI using the keyboard.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
```
Comment (chri2547) [inline _docs/style/focus-states.md]: Strike this...

File: _docs/style/focus-states.md [hunk 3]
```diff
@@ -3,29 +3,46 @@ title: Focus States
...
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
```
Comment (chri2547) [inline _docs/style/focus-states.md]: Strike this and sprinkle throughout.

File: _docs/style/focus-states.md [hunk 4]
```diff
@@ -34,358 +51,468 @@ Focus is received by interactive components when the user clicks on them.  Focus
 
 <section class="static-section" markdown="1">
 
-## Composition
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
```
Comment (chri2547) [inline _docs/style/focus-states.md]: Restructure to remove specifications.

File: _docs/style/focus-states.md (hunk 2 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done.

File: _docs/style/focus-states.md (hunk 3 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: n/a - decided to add a bp, see below.

File: _docs/style/focus-states.md (hunk 1 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done.

File: _docs/style/focus-states.md [hunk 5]
```diff
@@ -3,29 +3,36 @@ title: Focus States
...
 
-Focus is received by interactive components when the user clicks on them.  Focus is also applied to clickable elements when the user presses the tab key to traverse the UI using the keyboard.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% endcolumn %}
+## When to use
+
+Apply focus to all interactive elements in a UI.
+
+Do not focus UI components that do not provide interaction feedback, including:
```
Comment (palindromebart) [inline _docs/style/focus-states.md]: "UI components that do not provide interaction feedback, including:" is a confusing phrase. 

maybe change to 
"non-interactive elements, including:"

File: _docs/style/focus-states.md [hunk 6]
```diff
@@ -34,358 +41,454 @@ Focus is received by interactive components when the user clicks on them.  Focus
...
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Two visual features make up the focus indicator:
+### Composition
+
+Focus consists of the following elements:
 
-1. **The Glow** is always applied to the focused element.
-2. **The Border** is only applied to elements that already have a border.
+- **Glow (required)**: The glow is the ring that appears around the focused element.
```
Comment (palindromebart) [inline _docs/style/focus-states.md]: maybe call it a "hazy ring"? to distinguish it from the solid border line

File: _docs/style/focus-states.md [hunk 7]
```diff
@@ -34,358 +41,454 @@ Focus is received by interactive components when the user clicks on them.  Focus
...
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Standard focus color
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Standard focus color is **#0e94a6 (Cyan 700)**
+### Focused element with light fill or border
```
Comment (palindromebart) [inline _docs/style/focus-states.md]: these sections make MUCH more sense!!!!

File: _docs/style/focus-states.md (hunk 5 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done. woot!

File: _docs/style/focus-states.md (hunk 6 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Good one, and done!

File: _docs/style/focus-states.md (hunk 7 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: love. it.

File: _docs/style/focus-states.md [hunk 8]
```diff
@@ -34,358 +39,454 @@ Focus is received by interactive components when the user clicks on them.  Focus
...
 
-### Darker focus colors
+<div class="hxRow" markdown="1">
 
-If the element's fill or border color are darker than 500, apply that color to the glow and border instead of the standard focus color.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### For borderless elements:
-If the focused element's fill color is darker than 500, use that color for the glow.
+### Focused element with dark fill or border
```
Comment (chri2547) [inline _docs/style/focus-states.md]: add 'on light background'

File: _docs/style/focus-states.md [hunk 9]
```diff
@@ -34,358 +39,454 @@ Focus is received by interactive components when the user clicks on them.  Focus
...
+</section>
+
+<section class="static-section" markdown="1">
+
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Inside dark backgrounds
+### Focused element is on a dark background
```
Comment (chri2547) [inline _docs/style/focus-states.md]: add 'on light background'

**Comment by palindromebart** [comment]
Ship it, Squirrel!

File: _docs/style/focus-states.md [hunk 10]
```diff
@@ -3,424 +3,235 @@ title: Focus States
 layout: docs
 category: Style
 usage: |
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
usage: >
```

File: _docs/style/focus-states.md (hunk 4 above)
Comment (nimbinatus) [reply _docs/style/focus-states.md]: Is this comment resolved?

File: _docs/style/focus-states.md [hunk 11]
```diff
@@ -3,424 +3,235 @@ title: Focus States
...
 
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Buttons
+### Element with light fill or border
 
-In addition to primary, secondary and tertiary buttons, the following are a few elements that are also treated as buttons: 
+For elements without a border, if the element's fill color is 500 or lighter, use #0E94A6 (Cyan 700) for the glow color.
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]: I believe we've been putting hex codes and HTML colors in code fences elsewhere. Are we changing that? I don't want to point it out everywhere if that's not the case.

```suggestion
For elements without a border, if the element's fill color is 500 or lighter, use `#0E94A6` (`cyan 700`) for the glow color.
```

File: _docs/style/focus-states.md [hunk 12]
```diff
@@ -3,424 +3,235 @@ title: Focus States
...
 
-* [Radio Buttons]({{site.baseurl}}/components/radio-buttons.html)
-* [Checkboxes]({{site.baseurl}}/components/checkboxes.html)
-* [Actions Cog]({{site.baseurl}}/components/tables-controls.html#actions-cog)
+For elements with a border, if the element's border color is 500 or lighter, use #0E94A6 (Cyan 700) for both the glow and the border color.
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
For elements with a border, if the element's border color is 500 or lighter, use `#0E94A6` (`cyan 700`) for both the glow and the border color.
```

File: _docs/style/focus-states.md [hunk 13]
```diff
@@ -3,424 +3,235 @@ title: Focus States
...
 
-<section class="static-section" markdown="1">
-<div class="hxRow" markdown="1">
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Selector strips
+### Element is on a dark background
 
-For items in a selector strip, the text is the action driving element and is focused according to the rule for [Hyperlinks and Text](#hyperlinks-and-text).
+If the focused element is on a background color that is 500 or darker, change the glow and border to #FFFFFF (Gray 0).
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
If the focused element is on a background color that is 500 or darker, change the glow and border to `#FFFFFF` (`gray 0`).
```

File: _docs/style/focus-states.md [hunk 14]
```diff
@@ -3,424 +3,235 @@ title: Focus States
...
-
-<section class="static-section" markdown="1">
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Focusable containers
+### Element without visible boundaries
+
+If the focused element's boundary is not visible, use #0E94A6 (Cyan 700) for the glow color and add a 2px corner radius to elements that do not have a visible box shape, including:
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
If the focused element's boundary is not visible, use `#0E94A6` (`cyan 700`) for the glow color and add a 2px corner radius to elements that do not have a visible box shape, including:
```

File: _docs/style/focus-states.md [hunk 15]
```diff
@@ -3,424 +3,235 @@ title: Focus States
...
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
 ### Focusable child elements
 
-There are some instances in which a user can set focus on the child element of a focusable parent element and that parent does not lose focus.
+There can be cases when both a parent element and a child element receive focus simultaneously. To visually distinguish between the parent and child focus, the child focus is styled differently.
 
-Using the search input as an example, the clear button (child) takes on a dotted, 2px rounded border with the standard Cyan 700 focus color.
+For example, with the search filter, the clear button (child) takes on a dotted, 2px rounded border with the standard Cyan 700 focus color.
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
For example, with the search filter, the clear button (child) takes on a dotted, 2px rounded border with the standard `cyan 700` focus color.
```

File: _docs/style/focus-states.md (hunk 10 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done.

File: _docs/style/focus-states.md (hunk 4 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Yes, done.

File: _docs/style/focus-states.md (hunk 11 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Ah yes, ok, made that change throughout. Thanks!

File: _docs/style/focus-states.md (hunk 13 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done.

File: _docs/style/focus-states.md (hunk 12 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: done.

File: _docs/style/focus-states.md (hunk 14 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: done.

File: _docs/style/focus-states.md (hunk 15 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: Done.

**Pull request merged**